common/shlibs

```diff
@@ -3624,7 +3624,7 @@ libFAudio.so.0 FAudio-19.05_1
 libqaccessibilityclient-qt5.so.0 libqaccessibilityclient-0.4.0_1
 libnitrokey.so.3 libnitrokey-3.4.1_1
 libceres.so.1 ceres-solver-1.14.0_1
-libgraphene-1.0.so.0 graphene-1.8.2_1
+libgraphene-1.0.so.0 graphene-1.10.0_1
```

Comment:
is it really necessary to bump min version?

if true, why?

Reply:
I thought we were supposed to make sobumps each time we update libraries, sorry if thats not the case.

Reply:
nope unless:

- there's a SONAME bump
- a dependency needs symbols available in a recent version

Reply:
Well if thats the case I was working on a `mutter` update that needs the new symbols provided (at least `meson` enforces such version), but I didn't know that back then.

Reply:
Package version shouldn't need to be changed in common/shlibs unless the soname/symbols change.